test/kata5.reachDestination.test.js

```diff
@@ -1,5 +1,13 @@
 const { reachDestination } = require("../src");
 
 describe("reachDestination", () => {
-  test("returns string with estimated time of arrival", () => {});
+  test("returns string with estimated time of arrival", () => {
+    expect(reachDestination(44,10)).toEqual("I should be there in 4.5 hours.");
+    expect(reachDestination(60,15)).toEqual("I should be there in 4 hours.");
+    expect(reachDestination(120,11)).toEqual("I should be there in 11 hours.");
+    expect(reachDestination(46,7)).toEqual("I should be there in 6.5 hours.");
+  });
```

Comment:
Just indentation here, and a slight tweak to the last assertion as its a little bit out:
```
    test("returns string with estimated time of arrival", () => {
        expect(reachDestination(44, 10)).toEqual("I should be there in 4.5 hours.");
        expect(reachDestination(60, 15)).toEqual("I should be there in 4 hours.");
        expect(reachDestination(120, 11)).toEqual("I should be there in 11 hours.");
        expect(reachDestination(46, 7)).toEqual("I should be there in 7 hours.");
    });
```